src/app/settings/WishListSettings.tsx

```diff
@@ -116,6 +126,9 @@ export default function WishListSettings() {
             <button type="button" className="dim-button" onClick={clearWishListEvent}>
               <AppIcon icon={banIcon} /> {t('WishListRoll.Clear')}
             </button>
+            <button type="button" className="dim-button" onClick={reloadAllWishListsEvent}>
+              <AppIcon icon={refreshIcon} /> {t('WishListRoll.Refresh')}
+            </button>
```

Comment:
If GitHub really does cache for 10 minutes, we probably need to show a warning when this is clicked that explains why it might not be fresh.

Reply:
I am learning the patterns still, thought I could add a notification block in the handleReloadEvent right after the update completes reminding them as well as saying "we did it, promise"